dbt_platform_helper/domain/codebase.py

```diff
@@ -242,13 +244,6 @@ def __init__(self, codebase: str):
         super().__init__(f"""Your deployment for {codebase} was not triggered.""")
 
 
-class ApplicationEnvironmentNotFoundException(ApplicationException):
```

Comment:
This exception is used within maintenace page too so we moved it too a common place